src/api/helpers/map-CIS-countries/map-CIS-country/map-esra-classification/index.ts

```diff
@@ -0,0 +1,27 @@
+import { EXTERNAL_API_DEFINITIONS, EXTERNAL_API_MAPPINGS } from '../../../../constants';
+
+const { CIS } = EXTERNAL_API_DEFINITIONS;
+
+/**
+ * mapEsraClassification
+ * Transform a country's ESRAClassificationDesc field into a consistent string
+ * @param {String} Risk category
+ * @returns {String} Consistent ESRA classification
+ */
+const mapEsraClassification = (str: string): string | null => {
+  if (str === CIS.ESRA_CLASSIFICATION.STANDARD) {
```

Comment:
could we deference up to ESRA_CLASSIFICATION?

Reply:
I've removed the `CIS` part. I think anything more than this will make it confusing